modular_skyrat/modules/microfusion/code/cargo_stuff.dm

```diff
@@ -19,6 +19,12 @@
 		/obj/item/microfusion_phase_emitter/advanced,
 		/obj/item/stock_parts/cell/microfusion/advanced,
 		/obj/item/stock_parts/cell/microfusion/advanced,
+		/obj/item/microfusion_cell_attachment/rechargeable,
```

Comment:
This crate was changed when I removed the excess cells (you only needed one) so you might want to replace the single cells with bulk packs again.